bootcamp/tutorials/integration/integrate_with_phidata.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<p> </p><p> </p><p>To use Milvus, simply specify the collection name , <code>uri</code> and optionally <code>token</code>. Here is how to get the value of them:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>